Added changes to MATLAB, PT, and TF docs

matthewjreynolds opened the pull request:
I updated the MATLAB docs from CMS to reflect the newer version of MATLAB, changed multiple instances of "Eagle" to "Kestrel," and updated interactive and batch job commands. 

For PyTorch and TensorFlow documentation I updated the module name for anaconda.

yandthj commented on docs/Documentation/Applications/Matlab/batch.md:
```diff
@@ -53,6 +53,7 @@ Once the job has finished, the standard output is saved in a file called
 #SBATCH --nodes=1                # Number of nodes
 #SBATCH --job-name=matlabTest              # Name of job
 #SBATCH --account=<account_string>        # Program-based WCID (account string associated with job)
+#SBATCH --partition=standard
```
We're still recommending that users don't specify a partition in their job scripts unless using debug. Is there a specific reason for adding the standard partition?

matthewjreynolds replied:
I was getting weird behavior during my tests when I did not specify partitions, so I was adding them to be safe. Feel free to remove them.

matthewjreynolds replied:
I removed the specification of partition from the batch file and changed module load conda to module load anaconda3.

yandthj commented on docs/Documentation/Applications/Matlab/interactive.md:
````diff
@@ -10,11 +10,10 @@ Connection](https://www.nrel.gov/hpc/system-connection.html).
 
 ```
-$ srun -N # --time 01:00:00 --account=<handle> --pty $SHELL
+$ salloc --nodes=<number of nodes> --ntasks-per-node=<tasks per node> --account=<your account here> --time=<desired time> --partition=<desired partition>
````
We're still recommending that users don't specify a partition in their job scripts unless using debug. I think the partition flag could be removed.

matthewjreynolds replied:
Sae as above: the partition flag can be removed.

matthewjreynolds replied:
I removed the partition specification.

yandthj reviewed (changes requested):
Thanks for the updates! I just have a couple notes. I can make the suggested changes for you if you confirm they are correct: 

- Should the example job script on the Machine_Learning/index.md file also have "module load conda" changed to "module load anaconda3"? 
- Can I remove the --partition flags from the job submission commands? We still generally recommend not specifying the partition

matthewjreynolds commented:
Hi Haley,

I went through the MATLAB documentation and removed all the specifications of partition in the batch files and sallow commands. I also changed the machine learning index page so that: 1) the partition was not specified and 2) module load anaconda replaced module load conda.
